File: 2014/A.java
import java.util.Scanner;

public class A {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		double x = sc.nextInt();
		double y = sc.nextInt();
		
		double voltas;
		
		x = Math.abs(x-y);
		voltas = Math.ceil(y/x);
		
		//nao consegui tirar o ponto flutuante, estava dando erro java.util.IllegalFormatConversionException
		System.out.printf(voltas);
	}

}

File: 2014/A.cpp
#include <iostream>
#include<stdlib.h>
#include<math.h>
using namespace std;

int main(){
	double x, y;
	cin >> x >> y;
	double voltas;
	
	x = abs(x-y);
	voltas = ceil(y/x);
	
	cout << voltas << endl;
	return 0;
}
